containers-tests/tests/intmap-properties.hs

```diff
@@ -884,6 +886,9 @@ prop_intersectionWithKeyModel xs ys
 prop_disjoint :: UMap -> UMap -> Property
 prop_disjoint m1 m2 = disjoint m1 m2 === null (intersection m1 m2)
 
+prop_compose :: IMap -> IMap -> Int -> Property
+prop_compose bc ab k = (compose bc ab !? k) === (flip lookup ab >=> flip lookup bc) k
```

Comment:
Why not `((bc !?) <=< (ab !?)) k` to match the composition order and style? No big deal either way, of course.